cypress/global.d.ts

```diff
@@ -29,6 +30,7 @@ declare global {
             };
             mxDispatcher: MatrixDispatcher;
             mxPerformanceMonitor: PerformanceMonitor;
+            mxSettingsStore: SettingsStore; // to allow for adjusting settings in tests
```

Comment:
We can remove this before merging but this exercises the JSX support so we can see the errors in CI